### PR TITLE
[10.0][FIX] action server in tree view must play on all selected records

### PR DESCRIPTION
**Pull request by bguillot**
Hi,

The action server in tree view works only on the first selected record.

The PR fixes it.

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
LGTM

File: shopinvader_search_engine/views/product_view.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <field name="name">Recompute shopinvader product</field>
     <field name="model_id" ref="product.model_product_template"/>
     <field name="code">
-        for template in object:
+        for template in records:
```
Comment (lmignon) [inline shopinvader_search_engine/views/product_view.xml]: IMO this code should no work... `template` is never used

**Review by lmignon — CHANGES_REQUESTED** [review]

File: shopinvader_search_engine/views/product_view.xml (hunk 1 above)
Comment (bguillot) [reply shopinvader_search_engine/views/product_view.xml]: Yes thats right, it's luck that my patch fixed the bug ...
I will fix it.

**Comment by rousseldenis** [comment]
@bguillot

**Comment by rousseldenis** [comment]
@bguillot Any news ?

**Comment by sebastienbeau** [comment]
I have rebased the work and added a test so it's ready for merging

**Comment by rousseldenis** [comment]
@bguillot Could you fix conflict ?

**Comment by rousseldenis** [comment]
@lmignon Could you update your review ?

**Review by lmignon — APPROVED** [review]

**Pull request merged**